app/src/main/java/edu/mills/cs115/AZTermListFragment.java

```diff
@@ -94,8 +103,13 @@ public void onListItemClick(ListView l, View v, int position, long id) {
         getActivity().startActivity(intent);
     }
 
-    public void setCategory(int category) {
-        switch (category) {
+    /**
+     * Sets the string representation of a category by the position in which was selected.
```

Comment:
* Sets the string representation of a category by the position in which [it?] was selected.